### DISCUSSION TITLE
How I can build from source a package from official arch repos?

### DISCUSSION THREAD

**Discussion by PolpOnline** [Morganamilo/paru#413] [Ideas]
Hi, I would like to build from source a package that comes from the [extra] repo as it's a package from the AUR. 
I could clone the [svntogit-packages](https://github.com/archlinux/svntogit-packages) repo and use `paru -Ui` to install from the PKGBUILD, but the package won't automatically update.

**Comment by Morganamilo** [comment]
Paru can't auto update anything outside of the AUR.

**Reply by PolpOnline** [reply]
Can this become a feature? @Morganamilo

**Reply by Morganamilo** [reply]
Maybe some day. No plans right now.
